### PR TITLE
Change submodule, update upstream to v1.22 and add auto update script

### PR DESCRIPTION
**Pull request by vspinu**
CMD checked locally and build mlvocab with it. Everything looks good.

### CODE REVIEW

**Comment by vspinu** [comment]
Note that the current submodule link is one commit ahead of v1.22 which includes the exit+printf change which we need. In the future running `update.sh 1.23` should all the chores automatically.

File: .Rbuildignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 cran-comments.md
 inst/include/.git
 inst/include/.travis.yml
+update.sh
```
Comment (dselivanov) [inline .Rbuildignore]: new line please at the EOF

**Comment by dselivanov** [comment]
Thanks fa lot update and PR. If you don't mind - please put fix one minor comment I have. Then I can merge and submit to CRAN

**Comment by dselivanov** [comment]
Approved by Cran team, will be published soon - I think tomorrow.

**Comment by vspinu** [comment]
Great. Thanks!

**Pull request merged**